jmix-flowui/flowui/src/main/java/io/jmix/flowui/app/bulk/BulkEditView.java

```diff
@@ -0,0 +1,586 @@
+import com.vaadin.flow.router.Route;
+import io.jmix.core.DataManager;
+import io.jmix.core.EntitySet;
+import io.jmix.core.FetchPlan;
+import io.jmix.core.FetchPlanBuilder;
+import io.jmix.core.FetchPlanRepository;
+import io.jmix.core.FetchPlans;
+import io.jmix.core.Messages;
+import io.jmix.core.Metadata;
+import io.jmix.core.MetadataTools;
+import io.jmix.core.SaveContext;
+import io.jmix.core.annotation.TenantId;
+import io.jmix.core.entity.EntityValues;
+import io.jmix.core.metamodel.model.MetaClass;
+import io.jmix.core.metamodel.model.MetaProperty;
+import io.jmix.core.metamodel.model.MetaPropertyPath;
+import io.jmix.flowui.Dialogs;
+import io.jmix.flowui.Notifications;
+import io.jmix.flowui.UiComponents;
+import io.jmix.flowui.action.DialogAction;
+import io.jmix.flowui.action.DialogAction.Type;
+import io.jmix.flowui.bulk.BulkEditorDataProvider;
+import io.jmix.flowui.bulk.BulkEditorDataProvider.LoadDescriptor;
+import io.jmix.flowui.component.ComponentGenerationContext;
+import io.jmix.flowui.component.SupportsTypedValue;
+import io.jmix.flowui.component.SupportsValidation;
+import io.jmix.flowui.component.UiComponentsGenerator;
+import io.jmix.flowui.component.validation.ValidationErrors;
+import io.jmix.flowui.component.validation.Validator;
+import io.jmix.flowui.exception.ValidationException;
+import io.jmix.flowui.kit.action.Action;
+import io.jmix.flowui.kit.action.ActionPerformedEvent;
+import io.jmix.flowui.kit.action.ActionVariant;
+import io.jmix.flowui.kit.component.button.JmixButton;
+import io.jmix.flowui.util.OperationResult;
+import io.jmix.flowui.util.UnknownOperationResult;
+import io.jmix.flowui.view.ChangeTrackerCloseAction;
+import io.jmix.flowui.view.CloseAction;
+import io.jmix.flowui.view.DialogMode;
+import io.jmix.flowui.view.MessageBundle;
+import io.jmix.flowui.view.StandardOutcome;
+import io.jmix.flowui.view.StandardView;
+import io.jmix.flowui.view.Subscribe;
+import io.jmix.flowui.view.ViewComponent;
+import io.jmix.flowui.view.ViewController;
+import io.jmix.flowui.view.ViewDescriptor;
+import io.jmix.flowui.view.ViewValidation;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.metadata.BeanDescriptor;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static io.jmix.flowui.app.bulk.ColumnsMode.TWO_COLUMNS;
+
+@ViewController("bulkEditorWindow")
+@ViewDescriptor("bulk-edit-view.xml")
+@Route("bulk-edit")
+@DialogMode(resizable = true, width = "60em", height = "40em", minWidth = "30em")
```

Comment:
Odd view size. When defining both `width` and `heigh`, we use one of screen resolutions, e.g. 800x600, 1024x768, etc. (of course in `em`), thus following default aspect ratio 1.33. That's why the default list view `width` is `64em`. In your case, aspect ratio is 1.5. 

`64em` x `48em` is more correct choice.